amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/MixedAndIcebergTableDescriptor.java

```diff
@@ -92,10 +101,11 @@ public class MixedAndIcebergTableDescriptor extends PersistentBase
 
   private static final Logger LOG = LoggerFactory.getLogger(MixedAndIcebergTableDescriptor.class);
 
-  private final ExecutorService executorService;
```

Comment:
Maybe we should move this implementation to module `amoro-iceberg-format`?
If yes, we can do it in the next PR.

Reply:
yes